package-lock.json

```diff
@@ -357,9 +357,9 @@
       },
       "dependencies": {
         "fs-capacitor": {
-          "version": "6.2.0",
-          "resolved": "https://registry.npmjs.org/fs-capacitor/-/fs-capacitor-6.2.0.tgz",
-          "integrity": "sha512-nKcE1UduoSKX27NSZlg879LdQc94OtbOsEmKMN2MBNudXREvijRKx2GEBsTMTfws+BrbkJoEuynbGSVRSpauvw=="
+          "version": "2.0.4",
```

Comment:
Make sure to run `npm i` to restore fs-capacitor back to 6.2.0, and graphql-upload back to 11.0.0